Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Borgs/borgs_tools.yml

```
@@ -1,8 +1,9 @@
 - type: entity
-  name: epinephrine hypospray
   parent: BaseItem
-  description: A borg version of hypospray that automatically regenerates epinephrine.
   id: HyposprayBorgStandard
+  name: epinephrine hypospray
+  suffix: Robot, No Map
+  description: A borg version of hypospray that automatically regenerates epinephrine.
```

```suggestion
  description: A borg version of the hypospray that automatically regenerates epinephrine.
```

Maybe also change this to describe itself without having to refer to the hypospray, so new players can understand what it does without needing to know. A fast injector. Something along those lines. See what the hypo describes itself as.